### PR TITLE
Problem: Need a type for power meter

### PR DESCRIPTION
**Pull request by aquette**
Solution: Add it

Signed-off-by: Arnaud Quette <ArnaudQuette@Eaton.com>

### CODE REVIEW

**Comment by aquette** [comment]
Part of another PR, on fty-core... but wifi down, no more push tonight...

File: src/fty_common_asset_types.cc [hunk 1]
```diff
@@ -104,6 +106,9 @@ subtype_to_subtypeid (const std::string &subtype)
     else if(st == "sts") {
         return asset_subtype::STS;
     }
+    else if(st == "power-meter") {
```
Comment (EldoreiJK) [inline src/fty_common_asset_types.cc]: is dash a safe character?

Comment (aquette) [reply src/fty_common_asset_types.cc]: hey, that's a good question! ;)
I thought about converting, to be both more coherent with the other such assets (patchpanel, ...) and be sure WRT an extra dash, possibly used in some tokenization (not scanned the code to assert though).
Going to amend the needed PRs, thx for pointing @EldoreiJK

**Comment by aquette** [comment]
Closing in favor of https://github.com/42ity/fty-common/pull/92

**Pull request closed without merging**